src/components/card/card.stories.tsx

```diff
@@ -0,0 +1,512 @@
+)
+
+const mediaAndHeaderFirst = (
+  <Card gridLayout={{ tablet: { col: 4 } }} headerFirst>
+    <CardHeader>
+      <h2 className="usa-card__heading">Media and Header First</h2>
+    </CardHeader>
+    <CardMedia>
+      <img
+        src="https://images.unsplash.com/photo-1484100356142-db6ab6244067?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=962&q=80"
+        alt="A placeholder"
+      />
+    </CardMedia>
+    <CardBody>
+      <p>
+        {' '}
+        This is a standard card with the header and media first and a button in
+        the footer.{' '}
+      </p>
+    </CardBody>
+    <CardFooter>
+      <Button type="button" className="usa-button">
+        Example Button
+      </Button>
+    </CardFooter>
+  </Card>
+)
+
+const insetMedia = (
+  <Card gridLayout={{ tablet: { col: 4 } }} headerFirst>
+    <CardHeader>
+      <h2 className="usa-card__heading">Inset Media</h2>
+    </CardHeader>
+    <CardMedia inset>
+      <img
+        src="https://images.unsplash.com/photo-1484100356142-db6ab6244067?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=962&q=80"
+        alt="A placeholder"
+      />
+    </CardMedia>
+    <CardBody>
+      <p>
+        {' '}
+        This is a standard card with the header and media first, inset media,
+        and a button in the footer.{' '}
+      </p>
+    </CardBody>
+    <CardFooter>
+      <Button type="button" className="usa-button">
+        Example Button
+      </Button>
+    </CardFooter>
+  </Card>
+)
+
+const exdentMedia = (
+  <Card gridLayout={{ tablet: { col: 4 } }} headerFirst>
+    <CardHeader>
+      <h2 className="usa-card__heading">Exdent Media</h2>
+    </CardHeader>
+    <CardMedia exdent>
+      <img
+        src="https://images.unsplash.com/photo-1484100356142-db6ab6244067?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=962&q=80"
+        alt="A placeholder"
+      />
+    </CardMedia>
+    <CardBody>
+      <p>
+        This is a standard card with the header and media first, exdent media,
+        and a button in the footer.
+      </p>
+    </CardBody>
+    <CardFooter>
+      <Button type="button" className="usa-button">
+        Example Button
+      </Button>
+    </CardFooter>
+  </Card>
+)
+
+const exdentCard = (
+  <Card gridLayout={{ tablet: { col: 4 } }} headerFirst>
+    <CardHeader exdent>
+      <h2 className="usa-card__heading">Exdent Card</h2>
+    </CardHeader>
+    <CardMedia exdent>
+      <img
+        src="https://images.unsplash.com/photo-1484100356142-db6ab6244067?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=962&q=80"
+        alt="A placeholder"
+      />
+    </CardMedia>
+    <CardBody exdent>
+      <p>
+        This is a standard card with the header and media first, media, and a
+        button in the footer. All of which are exdent style.
+      </p>
+    </CardBody>
+    <CardFooter exdent>
+      <Button type="button" className="usa-button">
+        Example Button
+      </Button>
```

Comment:
I can't figure out what these footer and header exdent styles are supposed to show?  The button also looks weird. with these styles. Do we need to add this story example?

Reply:
I am not quite sure what the styles are suppose to implement either, but they are included in the USWDS documentation for the component, so I thought it best to include them still. I think it makes sense to have a story for them, given they are called out directly in the documentation.